### PR TITLE
Modified App Setup

### PR DESCRIPTION
**Pull request by jacqueline8711**
## Issue

This fixes the following issue(s): #742 

## Screenshot
screenshot_20180405-143638
screenshot_20180405-143642
screenshot_20180405-150246

## Why this is useful for all students
Easier/faster for users who don't want or can't log into TUMonline or for those that just don't want to enable the Token on the go

### CODE REVIEW

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/onboarding/LoginPromtCard.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import android.view.ViewGroup;
+
+import de.tum.in.tumcampusapp.R;
+import de.tum.in.tumcampusapp.component.ui.overview.CardManager;
+import de.tum.in.tumcampusapp.component.ui.overview.card.Card;
+import de.tum.in.tumcampusapp.utils.Const;
+import de.tum.in.tumcampusapp.utils.Utils;
+
+/**
+ * Card that describes how to dismiss a card
```
Comment (pfent) [inline app/src/main/java/de/tum/in/tumcampusapp/component/ui/onboarding/LoginPromtCard.java]: Nope, that's not what that card does :)

Maybe add the reasoning from #742 here?

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/onboarding/StartupActivity.java [hunk 2]
```diff
@@ -88,37 +87,6 @@ private void init() {
...
-        // For compatibility reasons: big update happened with version 35
-        int prevVersion = Utils.getSettingInt(this, Const.APP_VERSION, 35);
-
-        // get current app version
-        int currentVersion = Utils.getAppVersion(this);
-        boolean newVersion = prevVersion < currentVersion;
-        if (newVersion) {
-            this.setupNewVersion();
-            Utils.setSetting(this, Const.APP_VERSION, currentVersion);
-        }
```
Comment (pfent) [inline app/src/main/java/de/tum/in/tumcampusapp/component/ui/onboarding/StartupActivity.java]: what happened to `setupNewVersion()`?
I think we can either completely remove it then?

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/onboarding/WizNavCheckTokenActivity.java [hunk 3]
```diff
@@ -33,22 +33,9 @@ public void onCreate(Bundle savedInstanceState) {
...
-
-    /**
-     * If back key is pressed start previous activity.
-     */
-    @Override
-    public void onBackPressed() {
-        finish();
-        startActivity(new Intent(this, WizNavStartActivity.class));
-        overridePendingTransition(R.anim.fadein, R.anim.fadeout);
+    public void onBackPressed(){
```
Comment (pfent) [inline app/src/main/java/de/tum/in/tumcampusapp/component/ui/onboarding/WizNavCheckTokenActivity.java]: I think this is missing an `@Override`

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/onboarding/StartupActivity.java (hunk 2 above)
Comment (jacqueline8711) [reply app/src/main/java/de/tum/in/tumcampusapp/component/ui/onboarding/StartupActivity.java]: I'm not really sure, we probably can. What do you think @kordianbruck ?

Comment (kordianbruck) [reply app/src/main/java/de/tum/in/tumcampusapp/component/ui/onboarding/StartupActivity.java]: We can probably get rid of that as well. Just a question of if we already want to skip resetting the DB. I had to trigger a destructive migration for the current 1.5.6 release, as we were getting many crash reports from users.

**Pull request merged**